notebooks/Understand_Tables.ipynb

```diff
@@ -0,0 +1,4387 @@
+{
```

Comment:
<p>Minor nit - but I think there's a space before "Understand"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Tightened up the HTML at the top of this cell.</p>